core/gcc/version

```diff
@@ -1 +1 @@
-9.2.0 1
+9.2.0 2
```

Comment:
Keep the version the same. I'd rather not force a rebuild of GCC onto all users. GCC 10.0.0 is not too far away anyhow. Until then, those wanting OpenMP to work are able to manually run `kiss b gcc`. :+1: